### DISCUSSION TITLE
Missing Environment Variable, Retry Succeeds

### DISCUSSION THREAD

**Discussion by CalebC-RW** [community/community#26549] [Actions]
<p>Please explain what I am doing wrong.  It seems that at certain times, many of my build jobs fail complaining of a missing environment variable (SLACK_WEBHOOK).  Retrying each job succeeds.  This is the step that fails:</p>
<pre><code class="lang-auto">      - name: Slack notification for Open Tunnel
        env:
          SLACK_WEBHOOK: ${{ secrets.OPENTUNNEL_SLACK_WEBHOOK }}
        uses: Ilshidur/action-slack@2.1.0
        with:
          args: "message goes here"
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<p>Are the failing jobs triggered by pull requests? Pull requests from forks (and Dependabot) don’t have access to secrets.</p>

**Comment by CalebC-RW** [comment]
<p>Thanks for the thought, <a class="mention" href="/airtower-luna">@airtower-luna</a>.</p>
<p>The failing jobs are triggered by pull requests but the repo is not forked (and does not have forks).  The pull requests are from standard feature branches.  When I retry failed jobs, they succeed without any problem.  Additionally, most of the time the jobs succeed the first time.</p>
